packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -63,6 +63,13 @@ class AnalyzeCommand extends FlutterCommand {
         }
       }
 
+      String updaterPackagePath = path.join(ArtifactStore.flutterRoot, 'packages', 'updater');
```

Comment:
I meant we should check all flutter/packages/*/lib/*/main.dart files, the same way we check all flutter/packages/*/bin/*.dart files.